openerp/addons/base/res/res_company.py

```diff
@@ -229,13 +229,14 @@ def _company_default_get(self, cr, uid, object=False, field=False, context=None)
         if not context:
             context = {}
         proxy = self.pool.get('multi_company.default')
+        user = self.pool.get('res.users').browse(cr, SUPERUSER_ID, uid, context=context)
```

Comment:
Underscore is to limitate rpc, not to limitate the internal use. It is not really a "private" method.